README.adoc

```diff
@@ -130,38 +130,43 @@ TIP: If you are using https://pages.github.com/[GitHub Pages], plan ahead by kee
 
 === Prerequisites
 
-First you must install and configure {uri-nodejs-download}[Node.js] on your machine.
+First you must install and configure {uri-nodejs-download}[Node] on your machine.
 
 === Install
 
-Using npm:
+We recommend to install the dependencies in a project directory, such as the directory where your AsciiDoc presentations are stored.
+If you don't have a `package.json` file in your project directory, you can create one to eliminate warnings during the installation using:
 
-Create a directory to place slides. Initialize the directory to store npm packages within that directory. 
+ $ npm init -y
```

Comment:
Since `echo "{}" > package.json` does not work on Windows I think it's best to use `npm init -y`.